editor/src/components/inspector/sections/comment-section.tsx

```diff
@@ -193,57 +195,69 @@ const ThreadPreview = React.memo(({ thread }: ThreadPreviewProps) => {
       className={'thread-preview'}
       key={comment.id}
       onClick={onClick}
-      style={{
-        backgroundColor: isSelected
-          ? colorTheme.primary10.value
-          : thread.metadata.resolved
-          ? colorTheme.bg2.value
-          : 'transparent',
+      css={{
+        display: 'flex',
+        flexDirection: 'row',
+        paddingBottom: 5,
+        position: 'relative',
+        '&:hover': {
+          backgroundColor: isSelected ? colorTheme.primary10.value : colorTheme.bg2.value,
+        },
+        backgroundColor: isSelected ? colorTheme.primary10.value : 'transparent',
       }}
     >
-      <CommentWrapper
-        data-theme={theme}
-        user={user}
-        comment={comment}
-        showActions={false}
-        style={{ backgroundColor: 'transparent' }}
-      />
-      {when(
-        remixLocationRouteLabel != null,
+      <div
+        style={{
+          height: 46,
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+          justifyContent: 'center',
+        }}
+      >
         <div
           style={{
-            paddingLeft: 44,
-            paddingBottom: 11,
-            display: 'flex',
-            alignItems: 'center',
-            justifyContent: 'space-between',
+            width: 8,
+            height: 8,
+            borderRadius: 8,
+            position: 'relative',
+            left: 5,
+            background: readByMe === 'unread' ? colorTheme.primary.value : 'transparent',
           }}
-        >
+        />
+      </div>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          filter: thread.metadata.resolved ? 'grayscale()' : undefined,
+        }}
+      >
+        <CommentWrapper
+          data-theme={theme}
+          user={user}
+          comment={comment}
+          showActions={false}
+          style={{ backgroundColor: 'transparent' }}
+        />
+        {when(
+          remixLocationRouteLabel != null,
           <div
             style={{
+              paddingLeft: 44,
               fontSize: 9,
               color: colorTheme.fg6.value,
             }}
           >
-            {thread.metadata.remixLocationRoute ?? ''}
-          </div>
-        </div>,
-      )}
-      <div
-        style={{
-          paddingBottom: 11,
-          paddingLeft: 44,
-          paddingRight: 10,
-          marginTop: -5,
-          display: 'flex',
-          alignItems: 'center',
-          justifyContent: 'space-between',
-        }}
-      >
+            Route:{' '}
+            <span style={{ fontWeight: 500 }}>{thread.metadata.remixLocationRoute ?? ''}</span>
```

Comment:
Why are we not using `remixLocationRouteLabel` here (since we already know that is not `null`)? It seems like we're not using at all, so if that's deliberate we should just change the above `when` clause to be `thread.metadata.remixLocationRoute != null`